week1/.eslintrc.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
it seems like you have 2 `.eslintrc.json`, one is enough. You can have several eslint configuratons files if you want for example the project to follow some rules, but maybe tests should follow different rules.